### PR TITLE
feat: properly support Array.isArray.

### PR DESCRIPTION
**Pull request by mprobst**

### CODE REVIEW

**Comment by mprobst** [comment]
cc @vicb

File: test/facade_converter_test.ts [hunk 1]
```diff
@@ -92,6 +95,10 @@ describe('type based translation', () => {
     });
   });
 
+  it('translates array façades', () => {
+    expectWithTypes('var x = []; Array.isArray(x);').to.equal(' var x = [ ] ; ( x is List ) ;');
```
Comment (vicb) [inline test/facade_converter_test.ts]: should it be `( (x) is List )` ?

Comment (mprobst) [inline test/facade_converter_test.ts]: Good point, done.

**Comment by vicb** [comment]
LGTM

**Pull request merged**